src/test/java/org/jabref/model/entry/identifier/RFCTest.java

```diff
@@ -0,0 +1,37 @@
+package org.jabref.model.entry.identifier;
+
+import java.net.URI;
+import java.util.Optional;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class RFCTest {
+
+    @ParameterizedTest
+    @CsvSource({
+            "rfc7276, rfc7276",
+            "https://www.rfc-editor.org/rfc/rfc7276.html, rfc7276"
+    })
+    void testParseValidRfcIdAndUrl(String input, String expected) {
```

Comment:
Please be consistent.

assertEquals first gets expected, thus please, first `expected` then `input`

I fix it for myself. Too much context switches.